login-workflow/src/components/error-dialog/error-dialog.component.ts

```diff
@@ -5,10 +5,12 @@ import { ErrorDialogData } from '../../services/dialog/error-dialog.service';
 @Component({
     selector: 'pxb-auth-error-dialog',
     template: `
-        <div class="mat-h2">{{ title }}</div>
-        <div class="mat-subheading-2">{{ message }}</div>
-        <div style="display: flex; justify-content: flex-end; margin-top: 24px;">
-            <button mat-stroked-button (click)="dialogRef.close()">Okay</button>
+        <div style="max-width: 552px">
```

Comment:
meh... I'm guessing this was added for responsiveness purposes but can sometimes look really narrow on desktop. I can take it or leave it, just wanted to point it out.

Reply:
I think we should leave this in because I don't want to hard-code the width of the dialog, unless we have some specs to go off around what an appropriate min-width would be. I think for now, the length of the error-message can dictate the width of the box.